### PR TITLE
Fix 2 possible NullPointer occurences

### PR DESCRIPTION
**Pull request by mawulf**

### CODE REVIEW

File: src/main/java/com/gitblit/utils/FileUtils.java [hunk 1]
```diff
@@ -154,6 +155,14 @@ public static String readContent(File file, String lineEnding) {
 			System.err.println("Failed to read content of " + file.getAbsolutePath());
 			t.printStackTrace();
 		} finally {
+			if (reader != null){
```
Comment (gitblit) [inline src/main/java/com/gitblit/utils/FileUtils.java]: I don't believe closing the BufferedReader wrapper is necessary as the underlying stream is closed.

File: src/main/java/com/gitblit/manager/FederationManager.java [hunk 2]
```diff
@@ -363,6 +363,10 @@ public boolean accept(File file) {
 							&& file.getName().toLowerCase().endsWith(Constants.PROPOSAL_EXT);
 				}
 			});
+			if (files == null) {
```
Comment (gitblit) [inline src/main/java/com/gitblit/manager/FederationManager.java]: This is a nice catch.

Comment (mawulf) [inline src/main/java/com/gitblit/manager/FederationManager.java]: Yes but this returns the empty initialized list before files(null) can throw a NPE. Or Not?

Comment (gitblit) [inline src/main/java/com/gitblit/manager/FederationManager.java]: Your proposed change is good.  Returning an empty list is what I want, not throwing an NPE.

**Pull request merged**